costmap_2d/cfg/ObstaclePlugin.cfg

```diff
@@ -6,6 +6,8 @@ gen = ParameterGenerator()
 
 gen.add("enabled", bool_t, 0, "Whether to apply this plugin or not", True)
 gen.add("footprint_clearing_enabled", bool_t, 0, "Whether to clear the robot's footprint of lethal obstacles", True)
+gen.add("reduced_footprint_enabled", bool_t, 0, "Enable use of reduced footprint size for clearing lethal obstacle when footprint_clearing_enabled", False)
+gen.add("footprint_reduction_size", double_t, 0, "Size to reduce footprint (meters)", 0.01, 0.0, 0.1)
```

Comment:
These params are a bit confusing
Idea: add a single param named `footprint_clearing_padding` that defaults to `footprint_padding`
Setting to - footprint_reduction_size should achieve the same result
Ensure u add a good description, because the concept is not intuitive at all

Reply:
Hi did you mean like this?
there is only extra parameter now

https://github.com/user-attachments/assets/f26789e9-ff68-4017-86eb-7e2ba5a46e2d